src/bindings/python/nixl_bindings.cpp

```
@@ -629,7 +700,8 @@ PYBIND11_MODULE(_bindings, m) {
                const nixl_xfer_dlist_t &remote_descs,
                const std::string &remote_agent,
                const std::string &notif_msg,
-               std::vector<uintptr_t> backends) -> uintptr_t {
+               std::vector<uintptr_t> backends,
```

Cppcheck (2.19.0)

[performance] Function parameter 'backends' should be passed by const reference.

(passedByValue)